control/joy_controller/config/joy_controller.param.yaml

```diff
@@ -1,6 +1,6 @@
 /**:
   ros__parameters:
-    joy_type: $(var joy_type)
+    joy_type: DS4 # DS4, G29, P65, XBOX
```

Comment:
I think the `# DS4, G29, P65, XBOX` can be removed as these live in the schema:
* https://github.com/autowarefoundation/autoware.universe/blob/main/control/joy_controller/schema/joy_controller.schema.json#L13

Reply:
@xmfcx 
I don't think it's necessary but if you want to keep it, I'm OK with it!